pydropsonde/circles.py

```diff
@@ -219,6 +219,44 @@ def interpolate_na_sondes(self, method="cubic", max_gap=1500, thresh=4):
 
         return self
 
+    def remove_values(self, n_gap=3, keep_sfc=1000):
+        n_gap = int(n_gap)
+        ds = self.circle_ds
+
+        alt_mask = np.full(ds.u.shape, False)  # first sonde_id then alt_dim
+        alt_mask[:, ::n_gap] = True
+        if keep_sfc:
+            alt_mask[:, : int(keep_sfc / 10)] = True
+        for var in ["u", "v", "rh", "q", "ta", "theta", "x", "y"]:
```

Comment:
The division by `10` and the hardcoded list of variables seems pretty arbitrary. Thus I assume that this function was created with a specific application in mind (instead of something of generic use). But I can't find any site which uses this function, so there's also no specific application.